### DISCUSSION TITLE
JSX and fallthrough attributes causes type errors

### DISCUSSION THREAD

**Discussion by ThomasKientz** [vuejs/language-tools#3464] [General]
Adding an attribute ("extra" in the following exemple) does not raise any errors with a template but does raise an error with jsx. 

Am I doing something wrong or is a bug ? 

Extra is not declared as a prop in the `<HelloWorld>` component, it means to be used as a [fallthrough-attribute](https://vuejs.org/guide/components/attrs.html#fallthrough-attributes)

<img width="1108" alt="Capture d’écran 2023-08-02 à 12 03 04" src="https://github.com/vuejs/language-tools/assets/10157845/37e6d594-7dea-4fa7-9524-cad27354db42">

**Comment by so1ve** [comment]
#3301
